### ISSUE TITLE
Can't launch Desura if path contain accents

### ISSUE THREAD

**Issue by haedri** [desura/Desurium#91]
I've notice a small bug on Desura client for Linux

Under french Ubuntu the default download location is something like that : /home/_user name_/Téléchargements/
Which contains accent in the path, when extracting and installing from here you get following error after signing up :

jb@jb-desktop:~$ '/home/jb/Téléchargements/desura/desura'
terminate called after throwing an instance of 'sqlite3x::database_error'
  what():  unable to open database

There is also a popup error about a missing png when executing, that's what gave me the clue about the accent thingy (the png file is on the filesystem, and the displayed path is correct). And nothing happens when trying to register from the app (white window).

Moving the installation (or reinstalling) to another path without accent fix the issue

**Comment by Jookia** [comment]
This isn't something Desurium can fix, as the Desura client's installer does this. I'll leave the issue open for Protektor/Lodle to hack at.

**Comment by Protektor-Desura** [comment]
Are we saying that it doesn't handle directories with accent marks? If so that would effect all versions.

**Comment by haedri** [comment]
Indeed, I tested under Windows, same problem, but the error should less occur on Windows as default installation path doesn't contain accent mark (C:\Program Files\Desura).

**Comment by Protektor-Desura** [comment]
I thought this issue of directories with accent marks came up in beta testing and Iodle fixed it.

**Comment by lodle** [comment]
The issue was raised, never got around to fixing as boost and wx widgets handles it differently and could never work out how to get them to work together
